packages/core/src/components/index.scss

```diff
@@ -20,11 +22,14 @@
 @mixin components() {
   @include button.Button();
   @include checkbox.Checkbox();
-  @include field-label.FieldLabel();
   @include helper-text.HelperText();
   @include icon.Icon();
   @include input.Input();
   @include radio.Radio();
   @include search.Search();
   @include textarea.Textarea();
+
+  @if not includes.the('field-label') {
+    @include field-label.FieldLabel();
+  }
```

Comment:
I suppose we could hoist the include to the top and it will always execute before `Input` and `Textarea`, thus removing the need for the `@if` here.

What do you think?

```tsx
@include field-label.FieldLabel();

@include input.Input();
@include textarea.Textarea();
```

Reply:
Sure, just added little comment separating two lists.